apps/web/src/pages/Applications/ApplicantDashboardPage/components/TrackApplications/TrackApplicationsStatus.tsx

```diff
@@ -82,57 +83,62 @@ const QualifiedRecruitmentStatus = ({
           {intl.formatMessage({
             defaultMessage:
               "You are actively appearing in talent results for this recruitment.",
-            id: "TDikDt",
+            id: "5xZ350",
             description:
-              "Not suspended recruitment status message on qualified recruitment card.",
+              "Not suspended recruitment status message on application card.",
           })}
-          {/* Dialog that allows user to suspend the qualified recruitment (application). Sets suspendedAt date to the current time. */}
-          <QualifiedRecruitmentDialog
+          {/* Dialog that allows user to suspend the application. Sets suspendedAt date to the current time. */}
+          <TrackApplicationsDialog
             id={id}
             isSuspended={!!suspendedAt}
             title={intl.formatMessage({
               defaultMessage: "Change your appearance in search results",
-              id: "FGF3sd",
+              id: "3qdrpr",
               description:
-                "Dialog title on change search results status from qualified recruitment section.",
-            })}
-            primaryBodyText={intl.formatMessage({
-              defaultMessage:
-                "You are currently appearing in talent search results for the “IT-01 Helpdesk support technician” pool.",
-              id: "dPoE9z",
-              description:
-                "Dialog main body on change search results status from qualified recruitment section.",
+                "Dialog title on change search results status from applications section.",
             })}
+            primaryBodyText={intl.formatMessage(
+              {
+                defaultMessage:
+                  "You are currently appearing in talent search results for the “{poolName}” pool.",
```

Comment:
Another hard-coded message fix.

Reply:
I believe we made the decision to use the straight quotes in English and want to avoid `“”`.